projects/plugins/jetpack/changelog/update-block-validation-method

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: enhancement
+
+Block parsing tests: make call for main block settings more robust (no public changelog required)
```

Comment:
If this doesn't need a public changelog, would it be better to use the `Comment` header and leave the changelog itself blank?

```
Significance: patch
Type: enhancement
Comment: Block parsing tests: make call for main block settings more robust
```

Reply:
yeh, that would make more sense - done.